src/service/resources-impl.js

```diff
@@ -2725,3 +2718,12 @@ export let SizeDef;
 export function installResourcesServiceForDoc(ampdoc) {
   registerServiceBuilderForDoc(ampdoc, 'resources', Resources);
 }
+
+/**
+ * @param {!./ampdoc-impl.AmpDoc} ampdoc
+ */
+export function installOwnersServiceForDoc(ampdoc) {
+  registerServiceBuilderForDoc(ampdoc, 'owners', ampdoc => {
+    return Services.resourcesForDoc(ampdoc);
```

Comment:
Does this rely on Dima's change to make FIE an ampdoc?

Reply:
no